extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -221,6 +221,50 @@ public function getWikiVariables() {
 	}
 
 	/**
+	 * @desc Returns UA dimensions
+	 */
+	public function getTrackingDimensions() {
+		global $wgDBname, $wgUser, $wgCityId, $wgLanguageCode, $wgTitle;
+
+		$title = $wgTitle;
+		$article = Article::newFromID( $title->getArticleId() );
+
+		if ( $article instanceof Article && $title->isRedirect() ) {
+			$title = $this->handleRedirect( $title, $article, [] )[0];
+		}
+
+		$wikiCategoryNames = WikiFactoryHub::getInstance()->getWikiCategoryNames( $wgCityId );
+		$wikiCategoryNames = join( ',', $wikiCategoryNames );
+
+		$adContext = ( new AdEngine2ContextService() )->getContext( $title, 'mercury' );
```

Comment:
Oh, you're right! Sorry for not double checking. In that case I wouldn't bother ;)